### PR TITLE
Setting the download directory so you only get one instance downloaded…

### LINKED ISSUE #1080: setting instance_dir causes solr to download for each project
https://github.com/projecthydra/active_fedora/blob/master/lib/generators/active_fedora/config/solr/templates/solr_wrapper_test.yml#L4

@mark-dce @cam156 was this your intent?

### PR DESCRIPTION
**Pull request by cam156**
per machine. fixes #1080

### CODE REVIEW

File: lib/generators/active_fedora/config/fedora/templates/.fcrepo_wrapper [hunk 1]
```diff
@@ -2,3 +2,4 @@
 port: 8984
 enable_jms: false
 fcrepo_home_dir: tmp/fcrepo4-development-data
+download_dir: /tmp
```
Comment (jcoyne) [inline lib/generators/active_fedora/config/fedora/templates/.fcrepo_wrapper]: I wonder if these should be `<%= ENV['TMPDIR'] %>`? @cbeer ?

Comment (cbeer) [inline lib/generators/active_fedora/config/fedora/templates/.fcrepo_wrapper]: That'd certainly be better (and cross platform..)... although isn't this the same as the default behavior upstream?

Comment (mjgiarlo) [inline lib/generators/active_fedora/config/fedora/templates/.fcrepo_wrapper]: `<%= ENV.fetch('TMPDIR', '/tmp') %>` ?

Comment (jcoyne) [inline lib/generators/active_fedora/config/fedora/templates/.fcrepo_wrapper]: @cbeer Yes, that is curious.  Why is it downloading then?

**Comment by jcoyne** [comment]
@cam156 are you sure this fixes the issue?

Prior to using this change I see:

```
require 'solr_wrapper'
=> true
irb(main):002:0> SolrWrapper.instance(config: 'config/solr_wrapper_test.yml').config.instance_dir
=> "tmp/solr-test"
irb(main):003:0> SolrWrapper.instance(config: 'config/solr_wrapper_test.yml').config.download_dir
=> "/var/folders/9t/rygbnddx0b1ckw6tjs3m18qm0000gq/T"
```

So it appears as though it's already looking in the tmp directory for the download_dir.

**Comment by cam156** [comment]
@jcoyne looking at the default here: https://github.com/cbeer/solr_wrapper/blob/master/lib/solr_wrapper/configuration.rb#L58-L60
If you are in a rails app it uses tmp instead of .tmp
So depending on where you run the wrapper it behaves differently.  So I am not 100% sure, but I would believe it to work...

**Comment by jcoyne** [comment]
@cam156 Yes, I see that. I wonder if it makes more sense to just remove that logic from solr_wrapper?

**Comment by cam156** [comment]
@jcoyne I would be ok with that approach.

**Comment by mjgiarlo** [comment]
@cam156 @jcoyne so am I hearing that this will be fixed in solr_wrapper, and no related change is needed in AF? or?

**Comment by jcoyne** [comment]
I think this should get fixed in solr_wrapper. It's a broader issue than just AF.

**Comment by mjgiarlo** [comment]
OK. @cam156 should we close this issue?

**Comment by cam156** [comment]
@mjgiarlo Is there an issue on Solr wrapper?  I would leave it open until there is...

**Comment by mjgiarlo** [comment]
@cam156 There are 0 open issues on solr_wrapper, so no.

**Pull request closed without merging**